src/components/Slider/Slider.tsx

```diff
@@ -88,20 +91,22 @@ export const Slider: FC<SliderProps> = React.forwardRef(
             hideValue = false,
             id,
             included = true,
+            labelPosition = 'bottom',
             marks,
             min = 0,
             minLabel,
             max = 100,
             maxLabel,
             name,
             onChange,
-            readonly = false,
+            readOnly = false,
```

Comment:
is this a breaking change?

Reply:
readonly has not been published yet, so nope.